<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>556</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calculate</string>
  </property>
  <property name="windowIcon">
   <iconset resource="ic.qrc">
    <normaloff>:/cc/free-icon-calculator-8072290.png</normaloff>:/cc/free-icon-calculator-8072290.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow#MainWindow{
	background-color: rgb(2, 149, 253);
	background-color: rgb(220, 34, 90);
	background-color: rgb(24, 24, 24);
	/*bacground-image: url(:/img/.png);*/
}

QLineEdit{
	border: 1px solid black;
	/*background-color: rgb(167, 255, 99);*/
	border-radius: 12px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>480</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: 13pt &quot;Simplex_IV25&quot;;
font: 9pt &quot;Segoe UI&quot;;
background-color: rgb(73, 73, 73);
border-radius: 1em;
border: 1px solid black;
color: rgb(185, 185, 185);</string>
    </property>
    <property name="text">
     <string>Рассчитать</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>231</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: bold 20pt &quot;Sitka Small&quot;;
color: rgb(19, 25, 7);
color: rgb(253, 255, 254);
color: rgb(185, 185, 185);
/*background-color: rgb(255, 249, 166);
border-radius: 10%;*/</string>
    </property>
    <property name="text">
     <string>Длина
гипотенузы</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>410</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 12px;
background-color: rgb(217, 217, 217);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>80</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 12px;
background-color: rgb(217, 217, 217);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>0</y>
      <width>151</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: bold 22pt &quot;Sitka Small&quot;;
color: rgb(19, 25, 7);
color: rgb(185, 185, 185);
/*background-color: rgb(255, 249, 166);
border-radius: 10%;*/</string>
    </property>
    <property name="text">
     <string>Катет 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>310</y>
      <width>331</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: bold 20pt &quot;Sitka Small&quot;;
color: rgb(19, 25, 7);
color: rgb(253, 255, 254);
color: rgb(185, 185, 185);
/*background-color: rgb(255, 249, 166);
border-radius: 10%;*/</string>
    </property>
    <property name="text">
     <string>Высота
призмы</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>320</y>
      <width>241</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: bold 20pt &quot;Sitka Small&quot;;
color: rgb(19, 25, 7);
color: rgb(253, 255, 254);
color: rgb(185, 185, 185);
/*background-color: rgb(255, 249, 166);
border-radius: 10%;*/</string>
    </property>
    <property name="text">
     <string>Сумма ребер
 призмы</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 12px;
background-color: rgb(217, 217, 217);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>250</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: 13pt &quot;Simplex_IV25&quot;;
font: 9pt &quot;Segoe UI&quot;;
background-color: rgb(73, 73, 73);
border-radius: 1em;
border: 1px solid black;
color: rgb(185, 185, 185);</string>
    </property>
    <property name="text">
     <string>Рассчитать</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>0</y>
      <width>211</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: bold 22pt &quot;Sitka Small&quot;;
color: rgb(19, 25, 7);
color: rgb(253, 255, 254);
color: rgb(185, 185, 185);
/*background-color: rgb(255, 249, 166);
border-radius: 10%;*/</string>
    </property>
    <property name="text">
     <string>Катет 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>160</y>
      <width>261</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: bold 25pt &quot;Sitka Small&quot;;
font: 30pt &quot;Segoe UI&quot;;
color: rgb(19, 25, 7);
color: rgb(253, 255, 254);
color: rgb(185, 185, 185);
/*background-color: rgb(255, 249, 166);
border-radius: 10%;*/</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>251</width>
      <height>42</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: bold 25pt &quot;Sitka Small&quot;;
font: 30pt &quot;Segoe UI&quot;;
color: rgb(19, 25, 7);
color: rgb(253, 255, 254);
color: rgb(185, 185, 185);
/*background-color: rgb(255, 249, 166);
border-radius: 10%;*/</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>521</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="ic.qrc"/>
 </resources>
 <connections/>
</ui>
